daemon/daemon_main.go

```diff
@@ -868,6 +868,10 @@ func initEnv(cmd *cobra.Command) {
 
 	switch option.Config.DatapathMode {
 	case option.DatapathModeVeth:
+		if name := viper.GetString(option.IpvlanMasterDevice); name != "undefined" {
+			log.WithField(logfields.Device, name).
```

Comment:
Nit: probably we need to introduce `logfields.IpvlanMasterDevice` and then replace `logfields.Device` with it.

Reply:
Agree, will make this as a separate commit given this needs also change elsewhere.